Tribler/Main/vwxGUI/__init__.py

```diff
@@ -53,6 +53,10 @@
 SEPARATOR_GREY = wx.Colour(210, 210, 210)
 FILTER_GREY = wx.Colour(240, 240, 240)
 
+DOWNLOAD_POLICY_HIDDEN_120_FALLBACK = 0
```

Comment:
I think it's more appropriate to move those into `Tribler/Core`, for example `Tribler/Core/simpledef.py`.